src/main/java/com/htisolutions/poolref/viewModels/ResetPasswordNicknameViewModel.java

```diff
@@ -0,0 +1,9 @@
+package com.htisolutions.poolref.viewModels;
```

Comment:
These ViewModels should probably be in another directory called "securityQuestion"